js/views/PageNav.js

```diff
@@ -140,7 +141,8 @@ export default class extends View {
   }
 
   navSettingsClick() {
-    // activate settings modal here
+    new SettingsModal().render().open();
```

Comment:
That part is intended to be temporary, I just needed a way to trigger the settings modal.